azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -67,15 +70,6 @@
 public class HadoopSecurityManager_H_2_0 extends HadoopSecurityManager {
 
   /**
-   * TODO Remove duplicated constants from plugins.
-   *
-   * Azkaban plugins don't depend on a common submodule from which they both can inherit code. Thus,
-   * constants are copied around and any changes to the constant values will break Azkaban. This
-   * needs to be fixed as part of a plugin infrastructure implementation.
-   */
-  public static final String NATIVE_LIB_FOLDER = "azkaban.native.lib";
```

Comment:
Removing this parameter may break downstream plugins. I suggest we avoid this refactor. Just mark this deprecated for now.